src/custom/state/claim/hooks/index.ts

```diff
@@ -86,14 +86,34 @@ export type RepoClaims = RepoClaimData[]
 /**
  * Gets an array of available claim
  *
+ * By default, it'll filter out claims that are no longer available when the time window is closed
+ * It's possible to ignore the time window and return all unclaimed claims
+ *
  * @param account
+ * @param ignoreTimeWindow
  */
-export function useUserAvailableClaims(account: Account): UserClaims {
+export function useUserAvailableClaims(account: Account, ignoreTimeWindow = false): UserClaims {
```

Comment:
just curious, in which cases will we use the hook with the ignoreTimeWindow? 

Also, would it make sense to call it `includeExpiredClaims`

Reply:
I'm thinking in case we want to show an overview of all claims, even if some have been missed.
Or for testing and emulating the old behaviour.

And I'll rename the flag

Reply:
No longer applies, removed